python/tvm/relay/op/contrib/bnns.py

```diff
@@ -70,7 +73,7 @@ def partition_for_bnns(mod, params=None):
         ]
     )
 
-    return seq(mod)
+    return seq(mod, None)
```

Comment:
Is this one intentionally different from the other similar changes?

Reply:
No, my bad, and fixed.